packages/trace-viewer/src/ui/networkResourceDetails.css

```diff
@@ -16,9 +16,10 @@
 
 .network-request-details-tab {
   width: 100%;
+  height: 100%;
   user-select: text;
   line-height: 24px;
-  margin-left: 10px;
+  padding-left: 10px;
```

Comment:
```suggestion
  margin-left: 10px;
```

Reply:
I reverted this for now. While it seems like an issue (double spacing) its unrelated to the actual issue we are fixing here.